helm/templates/route.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: {{.Values.gabi.instance}}
+  annotations:
+    {{- if .Values.cert_manager.enable}}
+    cert-manager.io/issuer-kind: {{.Values.cert_manager.issuer.kind}}
+    cert-manager.io/issuer-name: {{.Values.cert_manager.issuer.name}}
+    {{- end}}
+    haproxy.router.openshift.io/timeout: {{.Values.openshift_router_timeout}}
```

Comment:
can we also port this feature to oc template? https://github.com/app-sre/gabi/blob/main/openshift/gabi.template.yaml, extract the whole `annotations` as a parameter.

Reply:
The idea is replacing the openshift template with the helm chart. We don't need to maintain 2 different systems to deploy this. 

I don't like the approach of setting this as a parameter because we'll need to put the "timeout" annotation to every target in the saas file. I also consider the helm approach to be cleaner in that aspect.